server/graphql/v2/mutation/AccountMutations.ts

```diff
@@ -80,6 +81,12 @@ const accountMutations = {
           throw new Forbidden();
         }
 
+        if (!req.remoteUser.isAdminOfCollective(account)) {
+          checkRemoteUserCanUseHost(req);
+        } else {
+          checkRemoteUserCanUseAccount(req);
+        }
```

Comment:
I don't really get this condition, why check `canUseHost` if admin and `canUseAccount` if not an admin?

Reply:
That makes sense, I did not check the code above. It's actually clear enough.